### DISCUSSION TITLE
How do I set the size of a single log entry?

### DISCUSSION THREAD

**Discussion by JokerDevops** [Kong/kong#12736] [Help]
When I print the log, I print the body content as well. The size of my body is 5m, but it doesn't print the whole log. So how do I set the size of a single log entry so that it prints out completely?

**Comment by chronolaw** [comment]
Kong runs on Nginx/OpenResty, there is  a hard-code 2048 bytes limitation for log, so we can not print out too long string in the log file.
```
// in ngx_log.h
#define NGX_MAX_ERROR_STR   2048
```

**Reply by chronolaw** [reply]
The more details from OpenResty: https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#ngxlog
